decentraland/unity-explorer
# feat: exotic rarity

Added new NFT rarity: exotic

## Test steps:
1. Open the backpack and check that the wearables are displayed with the correct colors
2.
